README.md

```diff
@@ -10,7 +10,10 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 
 ```js
 
-var superagent = require('superagent-use');
+var superagent = require('superagent');
+/* Pass the superagent object to superagent-use
+   in order to be provided with new `use` functionality */
+require('superagent-use')(superagent);
```

Comment:
> but it should be the same version in order to reuse superagent's one

Yes, so in this case defined as `^3.0.0`.

> What do you think about the option I suggested?

I don't feel anything for the optional extend. If someone wants to reuse the superagent instance, they can easily wrap them in a custom module, from where they also can apply the global `use`d modules.

Eg;

```js
var superagent = require('superagent-use')(require('superagent'));

superagent.use(function(request) {
  if(request.url[0] === '/') {
    request.url = prefix + request.url;
  }
  return request;
})

module.exports = superagent;